### PR TITLE
Parallelize Unit Tests

### PR DESCRIPTION
**Pull request by jkingry**
Seems like at least most NUnit fixtures can be run in parallel

Added assembly level attributes so that all fixtures run in parallel by default, can be disabled on a per-fixture basis. My tun all tests time went from 133 seconds to 67 seconds, 50% faster

### CODE REVIEW

File: src/GitVersionCore.Tests/AssemblyAttributes.cs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿using NUnit.Framework;
+
+//[assembly: Parallelizable(ParallelScope.Fixtures)]
```
Comment (asbjornu) [inline src/GitVersionCore.Tests/AssemblyAttributes.cs]: Did you mean to comment this?

File: src/GitVersionTask.Tests/AssemblyAttributes.cs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+﻿using NUnit.Framework;
+
+//[assembly: Parallelizable(ParallelScope.Fixtures)]
```
Comment (asbjornu) [inline src/GitVersionTask.Tests/AssemblyAttributes.cs]: And this?

File: src/GitVersionExe.Tests/AssemblyAttributes.cs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+﻿using NUnit.Framework;
+
+//[assembly: Parallelizable(ParallelScope.Fixtures)]
```
Comment (asbjornu) [inline src/GitVersionExe.Tests/AssemblyAttributes.cs]: And this?

**Comment by jkingry** [comment]
@asbjornu I messed up and committed the code I was running to double check the timing without the attributes.
Fixed that, and used `AssemblyInfo.cs` as the file for assembly attributes as I see that is what's being done in other projects in the solution.

**Comment by asbjornu** [comment]
@jkingry: I see. Unfortunately, bringing those back broke the build. Could you take a look at it, please?

**Comment by jkingry** [comment]
Yeah. I was afraid of that. Will look into it.

**Pull request closed without merging**